### PR TITLE
Fix google_notebooks_instance labels not being able to be updated

### PR DESCRIPTION
**Pull request by bcreddy-gcp**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug

```

### CODE REVIEW

**Review by hao-nan-li — COMMENTED** [review]
Is this PR ready for review?

**Comment by bcreddy-gcp** [comment]
We are waiting for the change to be rolled out in the backend. I can get back to you for a review once it is rolled out.

**Comment by hao-nan-li** [comment]
> We are waiting for the change to be rolled out in the backend. I can get back to you for a review once it is rolled out.

Sounds good.

**Comment by bcreddy-gcp** [comment]
Change rolled out in the backend. Could you re-run the tests?

**Review by hao-nan-li — APPROVED** [review]

File: mmv1/third_party/terraform/services/notebooks/resource_notebooks_runtime_test.go.erb [hunk 1]
```diff
@@ -25,6 +25,7 @@ func TestAccNotebooksRuntime_update(t *testing.T) {
 				ResourceName:            "google_notebooks_runtime.runtime",
 				ImportState:             true,
 				ImportStateVerify:       true,
+        ImportStateVerifyIgnore: []string{"labels", "terraform_labels"},
```
Comment (hao-nan-li) [inline mmv1/third_party/terraform/services/notebooks/resource_notebooks_runtime_test.go.erb:28]: Could you fix the indentation?

**Pull request merged**